### PR TITLE
[ZCS-3189] Update Zmdiaglog for 8.8

### PR DESCRIPTION
**Pull request by DinoPuller**
Update Zmdiaglog including configurations and log files for network-ng

### CODE REVIEW

File: src/libexec/zmdiaglog [hunk 1]
```diff
@@ -601,12 +601,22 @@ sub run() {
     );
     cp $_, "$destination/confs" for @conf_files;
 
+    mkdir("$destination/confs/network");
+    my @network_conf_files = (
+        "/opt/zimbra/conf/zextras/config",
```
Comment (fatkudu) [inline src/libexec/zmdiaglog]: I assume this is a directory.  Does it make sense to include that in the list of things to copy?
Presumably, if none of these files exist, that doesn't pose a problem.  The way I've currently installed a FOSS install, I don't have a `/opt/zimbra/conf/zextras` directory present.  Have you tested what happens under those circumstances?

Comment (DinoPuller) [reply src/libexec/zmdiaglog]: "/opt/zimbra/conf/zextras/config" is a file. Simply it's ignored in a FOSS environment, anyway i've added a check to test if Network-NG is preset.

**Comment by grishick** [comment]
@tmclane could you please merge this into 'develop'? Last time I merged a 3rd party PR I messed something up in history :)

**Comment by tmclane** [comment]
@grishick Sure thing. Sorry I just now saw the comment directed at me.

**Pull request merged**